packages/zstandard/test_zstandard.py

```diff
@@ -2,22 +2,24 @@
 from pytest_pyodide import run_in_pyodide
 
 
-# Ensure C extension is used, and then CFFI as fallback, error if neither
+# Ensure that tests are executed with both the C extension and the CFFI backend
+@run_in_pyodide(packages=["zstandard"])
+def pytest_generate_tests(selenium, metafunc):
+    if "zstd_backend" in metafunc.fixturenames:
+        metafunc.parametrize("zstd_backend", ["cext", "cffi"])
```

Comment:
I'm not sure what's going on here now. You should be able to use a single parametrized fixture for this. You shouldn't need to have a `pytest_generate_tests` or parametrize each test func individually.

Reply:
Ah, I was trying to use this guide for parametrization: https://pytest-with-eric.com/introduction/pytest-generate-tests/

Let me try with the code snippet you added below.